schema.graphql

```diff
@@ -38,49 +88,122 @@ enum Chain {
 }
 
 type RootGauge @entity {
```

Comment:
Let's also introduce `addedTimestamp` and `factory` attributes in the RootGauge entity